hidden-role-to-go-core/src/main/java/com/github/virtualgamers/hrtg/core/model/Party.java

```diff
@@ -0,0 +1,111 @@
+package com.github.virtualgamers.hrtg.core.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import com.github.virtualgamers.hrtg.core.model.listener.PartyListener;
+
+/**
+ * Represents a party, or a grouping of {@link User}s playing a {@link Game}
+ *
+ * @author CCThomas
+ *
+ */
+@Entity
+@EntityListeners(PartyListener.class)
+public class Party implements Serializable {
+    private static final long serialVersionUID = 3680839567940085207L;
+
+    @Id
+    String id; // Will Act as Party Code
+
+    @Column(name = "party_leader_user_id")
+    String partyLeaderUserId;
```

Comment:
doc?

Reply:
Doc for what?

Reply:
for what it means, eg that this is the user who started the party?

Reply:
Well The user who started the party is the init party Id. Support will be added to change party leader (like if the current leader has to leave). Really I should implement that support now.